### ISSUE TITLE
7 invalid XML files

### ISSUE THREAD

**Issue by umarbutler** [wragge/hansard-xml#3]
Hi,
First off, thanks for putting out this dataset, it has proved really useful.

I just wanted to flag that there are 7 XML files that are invalid. They are:
```python
['https://raw.githubusercontent.com/wragge/hansard-xml/master/hofreps/1974/19741015_reps_29_hor91c1.xml',
 'https://raw.githubusercontent.com/wragge/hansard-xml/master/senate/1966/19660824_senate_25_s32.xml',
 'https://raw.githubusercontent.com/wragge/hansard-xml/master/senate/1966/19660914_senate_25_s32.xml',
 'https://raw.githubusercontent.com/wragge/hansard-xml/master/hofreps/1920/19201117_reps_8_94.xml',
 'https://raw.githubusercontent.com/wragge/hansard-xml/master/hofreps/1960/19600531_reps_23_hor27.xml',
 'https://raw.githubusercontent.com/wragge/hansard-xml/master/hofreps/1920/19201125_reps_8_94.xml',
 'https://raw.githubusercontent.com/wragge/hansard-xml/master/hofreps/1920/19201116_reps_8_94.xml']
```

They seem to be due to 404 errors. Maybe they need to be rescrape or perhaps they don't exist? I tried searching for one specifically on APH but couldn't find it.

**Comment by wragge** [comment]
Thanks for letting me know! Looks like they're there, but the file names have changed. Also seems that there might have been some improvements since I last harvested, so I think I'll reharvest the lot just to be sure.

**Comment by wragge** [comment]
Eg: https://parlinfo.aph.gov.au/parlInfo/download/hansard80/hansardr80/1920-11-16/toc_unixml/19201116_reps_8_94_v7.xml;fileType=text%2Fxml

**Comment by wragge** [comment]
I reharvested all of 1901-1980. There have been a lot of changes, and it seems corrections are ongoing. Some of the files changed as I was in the process of harvesting! I'll try and keep an eye on things and do another complete reharvest in a month or two to catch more changes.

I've checked all the xml files and they all seem to be ok (they parse as xml and have a `<hansard>` element). Let me know if you find any more problems.
